### DISCUSSION TITLE
PiMP #2 - Podcast Ownership Verification

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#381] [General]
This is the discussion board for the <podcast:verify> tag and the work going on around it.

**Comment by redimongo** [comment]
I would love to understand why we need this tag? When all podcasts currently support the owner tag

```
<itunes:owner>
<itunes:name>NAME OF OWNER</itunes:name>
<itunes:email>EMAIL ADDRESS OF OWNER</itunes:email>
</itunes:owner>
```

Now we get spammed every single day by people who contact us, all because we have the owner tag in our podcast. Why would we need a "ownership" verification?

Reason we don't need it:

1. We have the ownership tag
2. A podcast owner might change podcast hosting providers
3. Someone might steal our podcast name (a verified tick won't stop this, if anything it could hurt the real owner)
4. It won't stop spam

Love to hear why we do need it.

**Comment by jamescridland** [comment]
Hi, @redimongo 

* [The `itunes:owner` tag is no longer supported by Apple](https://podcasters.apple.com/4115-technical-updates-for-hosting-providers).

* Removing the email address from RSS feeds 100% stops spam sent to those email addresses, since the email addresses are no longer harvested. Because of that, podcast hosts like Buzzsprout and Podbean have already [removed the `itunes:owner` tag from their RSS feeds](https://www.buzzsprout.com/new/75-fighting-back-against-email-spam).

The `podcast:verify` tag is closed and isn't going to happen - it was expanded and simplified into the `podcast:txt` tag, which is already being used to prove to "Platform X" that I own "Podcast Y". Here's how it works:
> "Thanks for wanting to get listed on Platform X. So we know you own Podcast Y, could you please add "A3F55GF" somewhere in your podcast? If you've space to enter a verification code with your podcast host, use that: otherwise, put it in a recent episode description. Once we see it there, we'll know you own Podcast Y, and it'll be yours here on Platform X."

Finally, there's nothing that can be used to stop a podcast "stealing your podcast name". This tag (and its successor) is designed to stop Joe Schmo pretending that he owns _This American Life_, and controlling its listing on Spotify, as one example.

Hope all that helps.

**Reply by redimongo** [reply]
The issue comes down to Podcast platforms like Spotify which still requiring podcasters get an email confirmation. At least that is what happens to us, when we add a new show to Spotify - Am I missing something.

**Reply by jamescridland** [reply]
> The issue comes down to Podcast platforms like Spotify which still requiring podcasters get an email confirmation. At least that is what happens to us, when we add a new show to Spotify - Am I missing something.

No, you're not missing something. But, as more podcast hosting providers remove `itunes:owner` entirely, following Buzzsprout and Podbean's lead, they'll use the new verification code mechanism for this in future.
